packages/relay/src/lib/clients/clientCache.ts

```diff
@@ -84,9 +84,9 @@ export class ClientCache {
         let value = this.cache.get(key);
         if (value) {
             if (callingMethod) {
-                this.cacheKeyCounter.labels(key, callingMethod).inc(1);
+                this.cacheKeyGauge.labels(key, callingMethod).inc(1);
             } else {
-                this.cacheKeyCounter.labels(key, null).inc(1);
+                this.cacheKeyGauge.labels(key, '').inc(1);
             }
```

Comment:
Here Gauge is being used like a counter, no big issue, but might not be ideal.
Also, this could be simplified for readability to:

```
this.cacheKeyGauge.labels(key, callingMethod || '').inc(1);

```

Reply:
Yeah it's a supported method of Gauge.
Let me know why it would not be ideal?

Reply:
Adopted the simplification suggestion

Reply:
you are right, I looked and indeed is a supported method, I guess there is no problem.
Just that when I imagine a gauge I think on something that goes up and down and has some thresholds. like memory, cpu, temperature, active connections, etc... but to be honest, I don't see an issue either. 👍